model/metrics/system-metrics.yaml

```
@@ -248,6 +251,19 @@ groups:
       - ref: system.filesystem.mode
       - ref: system.filesystem.mountpoint
 
+  - id: metric.system.filesystem.limit
+    type: metric
+    metric_name: system.filesystem.limit
+    stability: experimental
+    brief: "The total storage capacity of the filesystem"
+    instrument: updowncounter
+    unit: "By"
+    attributes:
+      - ref: system.device
```

I know it's been like this before this PR, but I don't really understand what this attribute represents. Can we give it a better description or at least a good example?

---

This attribute value depends on the signal which is being used, for example, in system network metrics it will be equal to the network's interface while in system file system metrics it will be set to a disk identification/name. Would a single example be enough?

---

You can provide new brief/description that adds context/specialization to the attribute when it's referenced. E.g. here it's referenced under `system.filesystem.limit` and we could do something like

```suggestion
      - ref: system.device
         brief: The name of the disk.
```

---

Great, I was not aware of this feature. Added in https://github.com/open-telemetry/semantic-conventions/pull/1380/commits/f9deb24349f9a09a3809adf3e116e5ff25544106